package.json

```diff
@@ -5,6 +5,7 @@
   "repository": "zenorocha/clipboard.js",
   "license": "MIT",
   "main": "dist/clipboard.js",
+  "module": "src/clipboard.js",
```

Comment:
This change is technically a BC break, as `src/clipboard.js` is not only an ES module (which is wanted here) but is also not transpiled with Babel (which means that downstream project will need to configure their babel to avoid ignoring `node_modules/clipboard` if they need to support older browsers)

Reply:
That what `module` is intended to, point to es module. I'm agree that it is breaking change.

Reply:
ES module does not mean that it will be using other new ES features, that's independent. The BC break is not that it uses `export` and `import`, but that it uses `class` and other features.

Note however that I'm not against doing such BC break, but it needs to be accepted by the maintainer and released as a major version.

Reply:
I think i will remove this change for now, my main goal was to  reduce package size.